### DISCUSSION TITLE
Automation to play album from here

### DISCUSSION THREAD

**Discussion by acidcoke** [music-assistant/support#3050] [Q&A] [answered]
This is possible in the gui by using the button
![Play Album from here](https://github.com/user-attachments/assets/86ec9ffa-29ff-45f5-9fa9-d7ed4e6dc402)

Is it somehow possible to recreate this feature i.e. play an album starting from a certain track with an automation?

**Comment by OzGav** [comment, marked as answer]
No sorry not at this time.

**Reply by acidcoke** [reply]
Ok, Thx. 

Could you then give me some hints how I could implement this feature myself?

Edit: Would this need to be added to the server, the hass integration, or both?

**Reply by OzGav** [reply]
Yes I would expect both. The action would have to be changed to allow a track number option (but only for albums!) and then server needs to understand the request.
